src/SearchParameters/index.js

```diff
@@ -565,11 +584,16 @@ SearchParameters.prototype = {
    */
   removeNumericRefinement: function(attribute, operator, paramValue) {
     if (paramValue !== undefined) {
-      var paramValueAsNumber = valToNumber(paramValue);
```

Comment:
Is it expected that we dropped this conversion?

Reply:
yes, isNumericRefined already calls `valToNumber` on the value: https://github.com/algolia/algoliasearch-helper-js/pull/656/files#diff-ecd72976141723d9448df45e667cb4beR1224